### DISCUSSION TITLE
Pass function to action cell

### DISCUSSION THREAD

**Discussion by Soif2Sang** [shadcn-ui/ui#1080] [Q&A]
So my table is basically a user table and action column give me the possibility to delete or modify a user.
I want to fetch all the users again by giving a function to the cell and update the table data.

How to achieve this ?

**Comment by merRen22** [comment]
I had a similar problem when I wanted to add actions to my table. This [video](https://www.youtube.com/watch?v=Pt3-5aPu1pE) helps and also has a repo you can [check](https://github.com/CodeWithOsvaldas/finance-dashboard/blob/main/src/pages/bank-accounts/bankAccountsColumns.tsx)

**Reply by kyusungpark** [reply]
@merRen22 Hey thank you for your tutorial. I did something similar [here](https://github.com/shadcn-ui/ui/discussions/5176#discussion-7265582) , but using Sheets instead of a Modal.
I am running into a problem where when I move my mouse, it hightlights all the text in the input box. Also, it's not closing the dropdown when I click on the dropdown item. Any ideas why this might be happening?
